### PR TITLE
Feature 72 settings page

### PR DESCRIPTION
**Pull request by navina321**
completed settings page with testing

### CODE REVIEW

File: src/components/SettingsCard/SettingsCard.jsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+const Settings = () => {
+  return (
+    <main className="settings-card-container">
+      <div className="settings-card">
+        <div className="settings-card__content">
+          <section className="settings-card__mode">
+            <h1 className="settings-card__option">Light/Dark Mode:</h1>
+            <div className="settings-card__toggle-container">
+              <img className="settings-card__toggle-sun" src={Sun} alt="Sun" />
+
```
Comment (TristanPere) [inline src/components/SettingsCard/SettingsCard.jsx:16]: white space

File: src/components/SettingsCard/SettingsCard.jsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+              <label className="settings-card__toggle-switch">
+                <input type="checkbox" id = "toggle-switch"/>
+                <span className="settings-card__toggle-slider round"></span>
+              </label>
+
```
Comment (TristanPere) [inline src/components/SettingsCard/SettingsCard.jsx:21]: whitespace

File: src/components/SettingsCard/SettingsCard.jsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                src={Moon}
+                alt="Moon"
+              />
+            </div>
+          </section>
+          <section className="settings-card__theme">
+            <h1 className="settings-card__option">Select Theme:</h1>
+            <div className="settings-card__checkbox-container">
+              <div className="settings-card__modern">
+
```
Comment (TristanPere) [inline src/components/SettingsCard/SettingsCard.jsx:33]: whitespace

File: src/components/SettingsCard/SettingsCard.test.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { BrowserRouter } from "react-router-dom";
+import SettingsCard from "./SettingsCard";
+
+it("should render the apply button on the settings card on the page", () => {
+  //arrange
```
Comment (TristanPere) [inline src/components/SettingsCard/SettingsCard.test.js:7]: Moving forward please remove comments in test file of act arrange assert.

File: src/pages/Settings/Settings.jsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+        <div className="settings-page__content">
+          <SettingsCard />
+          <div className="settings-page__button-container">
+            <Button
+              className="settings-page__button"
+              buttonText="Log out"
+              buttonStyle="isLogout"
+            />
+          </div>
+
```
Comment (TristanPere) [inline src/pages/Settings/Settings.jsx:24]: white space

**Review by TristanPere — CHANGES_REQUESTED** [review]
The page looks great and matches figma well. However, the repo tests are failing as there is a use of start instead of flex start somewhere in your code (look at the PR conversation for more details on failed tests). Other notes are the settings buttons are not able to be unclicked and both can be simultaneously active when only one should be. Please make these changes along with the ones commented on the code.

**Comment by Joshua-Tucker** [comment]
We've made our changes based on navcontainer / homebutton and layout media queries changing max-width to 767px.

File: src/components/SettingsCard/SettingsCard.jsx [hunk 6]
```diff
@@ -0,0 +1,79 @@
+import React, {useState} from "react";
+import "./SettingsCard.scss";
+import Button from "../Button/Button";
+import Sun from "../../assets/images/lightmode/sun-light.png";
+import Moon from "../../assets/images/lightmode/moon-light.png";
+
+
```
Comment (hannahmitchell99) [inline src/components/SettingsCard/SettingsCard.jsx:7]: white space

**Review by hannahmitchell99 — CHANGES_REQUESTED** [review]

**Review by hannahmitchell99 — APPROVED** [review]

**Review by olirob93 — APPROVED** [review]

**Pull request merged**